pkg/apis/build/v1alpha2/build_types.go

```diff
@@ -69,20 +69,26 @@ type BuildSpec struct {
 	Notary    *NotaryConfig               `json:"notary,omitempty"`
 }
 
+type BuildCacheConfig struct {
+	VolumeName string `json:"volumeName,omitempty"`
+	ImageTag   string `json:"imageTag,omitempty"`
+}
+
 // +k8s:openapi-gen=true
 type Bindings []Binding
 
 // +k8s:openapi-gen=true
 type Binding struct {
-	Name        string                       `json:"name",omitempty"`
+	Name        string                       `json:"name,omitempty"`
 	MetadataRef *corev1.LocalObjectReference `json:"metadataRef,omitempty"`
 	SecretRef   *corev1.LocalObjectReference `json:"secretRef,omitempty"`
 }
 
 // +k8s:openapi-gen=true
 type LastBuild struct {
-	Image   string `json:"image,omitempty"`
-	StackId string `json:"stackId,omitempty"`
+	Image      string `json:"image,omitempty"`
+	CacheImage string `json:"cacheImage,omitempty"`
```

Comment:
To this point we have tried to keep every reference in the api to an image to use the `image` key. Could we move this to `cache.image`?